python/vtool/qt_ui.py

```diff
@@ -1131,8 +1089,7 @@ def create_branch(self, name=None):
         if current_item:
             self._add_sub_items(current_item)
             self.setItemExpanded(current_item, True)
-
-        if not current_item:
+        else:

```

Comment:
This may have broken something. As current item is assigned towards the later part of the first block.

Reply:
Ignore, it was the one on 1081.